doc/glossary.rst

```diff
@@ -267,6 +267,15 @@ If you wish to add a missing term, please `create a new issue`_ or
         `Support vector machines`_ are a set of :term:`supervised learning` methods used
         for :term:`classification`, :term:`regression` and outliers detection.
 
+    TFCE
+        Threshold-free cluster enhancement is a method for thresholding statistical maps
```

Comment:
I would call it a statistic that combines signal magnitude and cluster extent to enhance the importance of large and high clusters.

Reply:
Updated in [3bbdcaf](https://github.com/nilearn/nilearn/pull/3196/commits/3bbdcaf841351c108f57407572b0b6705500b29e).